test/react-native/features/fixtures/android-module/scenarios/BreadcrumbsNativeManualScenario.kt

```diff
@@ -0,0 +1,13 @@
+package com.<ANDROID_PACKAGE_PATH>.scenarios
```

Comment:
If you rebase the branch now that the pipeline refactor has gone in, you'll see that this package name can (needs to) now be explicitly set.